db/migrate/20220525183015_create_active_storage_tables.active_storage.rb

```diff
@@ -0,0 +1,36 @@
+# This migration comes from active_storage (originally 20170806125915)
+class CreateActiveStorageTables < ActiveRecord::Migration[5.2]
+  def change
+    create_table :active_storage_blobs do |t|
```

Comment:
why are the tables called "active_storage"? Please use a different name for the tables. It should be something that makes sense for the app.

Reply:
Hi There!, We were seeking about change active storage table name, but we realize about the command we ran to create the blob a active storage stuff with "rails active_storage:install" command creates a default name for the class and tables